homepairs/homepairsUI/src/Routes/RemoteEndpoints.tsx

```diff
@@ -384,38 +419,38 @@ export const postUpdatedAppliance = (
                     displayError(error);
                 }
             })
-            .catch(() => {});
+            .catch(error => console.log(error));
     };
 };
 
-export const postNewServiceRequest = (
+export const postNewServiceRequest = async (
```

Comment:
yay for the fix! Comments in this file could be a bit better / avoid this problem in the future